tests/api.c

```diff
@@ -13145,11 +13146,22 @@ static int test_wc_AesCbcEncryptDecrypt (void)
         if (cbcE == BAD_FUNC_ARG) {
             cbcE = wc_AesCbcEncrypt(&aes, enc, NULL, sizeof(vector));
         }
+#ifdef WOLFSSL_AES_CBC_ALIGNMENT_CHECKS
+        if (cbcE == BAD_FUNC_ARG) {
+            cbcE = wc_AesCbcEncrypt(&aes, enc, vector, sizeof(vector) - 1);
+        }
+        if (cbcE == BAD_ALIGN_E) {
+            cbcE = 0;
+        } else {
+            cbcE = WOLFSSL_FATAL_ERROR;
+        }
+#else
         if (cbcE == BAD_FUNC_ARG) {
```

Comment:
Instead of `#else` here it should be above `WOLFSSL_AES_CBC_ALIGNMENT_CHECKS` to keep existing check in all cases. These are two different unit tests.

Reply:
fixed.